grafeas-client/README.md

```diff
@@ -0,0 +1,69 @@
+# Ruby Client for Container Analysis API ([Alpha](https://github.com/googleapis/google-cloud-ruby#versioning))
+
+[Container Analysis API][Product Documentation]:
+An implementation of the Grafeas API, which stores, and enables querying and
+retrieval of critical metadata about all of your software artifacts.
+- [Client Library Documentation][]
+- [Product Documentation][]
+
+## Quick Start
+In order to use this library, you first need to go through the following
+steps:
+
+1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+2. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+3. [Enable the Container Analysis API.](https://console.cloud.google.com/apis/library/grafeas.googleapis.com)
+4. [Setup Authentication.](https://googleapis.github.io/google-cloud-ruby/#/docs/google-cloud/master/guides/authentication)
+
+### Installation
+```
+$ gem install grafeas
```

Comment:
Shouldn't this be `gem install grafeas-client`?

Reply:
Yes, good catch!